gradle.properties

```diff
@@ -14,7 +14,7 @@
 
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=1.2.4-SNAPSHOT
+version=1.3.0-SNAPSHOT
```

Comment:
You need to update the distribution to `2201.10.0` as well: https://github.com/ballerina-platform/module-ballerina-observe/blob/783e02115777bc5af10567c3290dc4dd245a3316/build-config/resources/Ballerina.toml#L5

Update the `ballerinaLangVersion` as well. You can use this timestamped version - `2201.10.0-20240802-124900-0b8d2ff7`

Reply:
Updated the lang version